docs/reference/supported-environments.md

```diff
@@ -4,6 +4,12 @@ title: "Supported environments"
 description: "Find out where to run Camunda 8 components for SaaS and Self-Managed, including Optimize for both Camunda 8 and Camunda 7."
 ---
 
+The supported environments page lists tested and compatible environments, deployment options, and component dependencies for successfully running Camunda 8.
+
+**If the particular technology is not listed, we are unable to resolve issues or provide guidance on integration.** You may [raise a feature request](/contact) that will be evaluated by our product teams to provide official support from Camunda.
```

Comment:
Should we re-use the wording from bevore for clarity?
```suggestion
**If you are running Camunda 8 using an environment, deployment option, or component dependency that is not listed, we cannot resolve issues or provide guidance on integration.** You may [raise a feature request](/contact) that will be evaluated by our product teams to provide official support from Camunda.
```

Reply:
Make sense from a consistency, but do we define environment, deployment option or component dependency anywhere? A customer will what to know what we mean. If we keep it as 'technology' we can use this a catch all which would be my preference instead of having to maintain more definitions

Reply:
The idea of this is to re-use the wording that is part of the page. The page would thereby give the definition. E.g., 

`Deployment Option`=> What a deployment option is, is defined as by the sub section of this page, which talks about `Deployment Options: HELM, Docker-compose, ...`.

@akeller can you please try to adjust this statement so that we pick up the terms of the sub-sections?

Reply:
"The supported environments page lists browsers, operating systems, clients, deployment options, and component requirements, which are tested and supported for compatibility with Camunda 8."